apps/src/templates/Button.jsx

```diff
@@ -116,7 +120,10 @@ class Button extends React.Component {
     }
 
     const sizeClassNames = __useDeprecatedTag
-      ? moduleStyles[size]
+      ? [
+          moduleStyles[size],
+          Phase1ButtonColor[color] ? moduleStyles.phase1Updated : ''
+        ]
       : [moduleStyles[size], moduleStyles.updated];
```

Comment:
Want to make sure I follow the logic here - what happens if `__useDeprecatedTag` is false, but `color` exists in `Phase1ButtonColor`? Do we not want to apply the `phase1Updated` class in this case?

Reply:
yeah, correct. Looks like the issue is only happens when we use __useDeprecatedTag. For regular buttons looks like everything works fine already